### PR TITLE
Make artillery wait for a url if specified

### PR DESCRIPTION
**Pull request by easternbloc**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,6 +1,16 @@
 #! /bin/bash
 set -e
 
+if [[ -n "${WAIT_URL}" ]]; then
+  echo "Waiting for ${WAIT_URL} to become available"
+  eval curl -fs "${WAIT_URL}" &> /dev/null;
```
Comment (daniel-ac-martin) [inline entrypoint.sh]: Do you need the output redirection with the `-s` flag?

File: entrypoint.sh [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+  while [ $$? -ne 0 ]; do \
```
Comment (daniel-ac-martin) [inline entrypoint.sh]: You might be able to do `until curl ...`.

**Comment by daniel-ac-martin** [comment]
Otherwise LGTM.

I suspect the clean up should be a PR on another repo too!

(I can do that it you remind me though.)

**Comment by easternbloc** [comment]
@daniel-ac-martin

**Pull request merged**